### ISSUE TITLE
Address accessibility issues in dev branch

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#530]
Based on our meeting with Dan to review his navigability of the dev GEAR site, please address the following issues: 

- [ ] The screen reader says “main region” for the left sidenav, but there needs to be more descriptive text there to indicate what the left sidenav is/does.
- [ ]  There is no indication there the user should select the menu or take any action on the left sidenav. The screen reader should indicate that the sidenav is collapsed or expandable so that the user knows to spacebar/enter into it.
- [ ] At the bottom of the dev homepage, there are 2 GSA images (one centered and one in the identifier component. Both of these should have descriptive text added to them (like "GSA logo").
